cocotb/_build_libs.py

```diff
@@ -305,6 +330,8 @@ def get_ext():
 
     logger.info("Compiling interface libraries for cocotb ...")
```

Comment:
For a future PR: These log lines aren't accurate, all you're doing is producing a list of libraries to be compiled.